src/pretix/static/pretixcontrol/js/ui/dragndroplist.js

```diff
@@ -1,48 +1,46 @@
 /*global $, Sortable*/
 $(function () {
     $("[data-dnd-url]").each(function(){
-        var container = $(this),
+        const container = $(this),
             url = container.data("dnd-url"),
             handle = $('<span class="btn btn-default btn-sm dnd-sort-handle"><i class="fa fa-arrows"></i></span>');
 
         container.find(".dnd-container").append(handle);
-        if (container.find("[data-dnd-id]").length < 2) {
+        container.find(".sortable-up, .sortable-down").addClass("sr-only");
+        if (container.find("[data-dnd-id]").length < 2 && !container.data("dnd-group")) {
             handle.addClass("disabled");
             return;
         }
-
+        let didSort = false, lastClick = 0;
+        function handleMouseUp() {
+            if (Date.now() - lastClick < 6000) {
+                container.find(".sortable-up, .sortable-down").removeClass("sr-only");
+            }
+            lastClick = Date.now();
+        }
+        container.find(".dnd-sort-handle").on("mouseup", handleMouseUp);
         Sortable.create(container.get(0), {
             filter: ".sortable-disabled",
             handle: ".dnd-sort-handle",
+            group: container.data("dnd-group"),
             onMove: function (evt) {
                 return evt.related.className.indexOf('sortable-disabled') === -1;
             },
             onStart: function (evt) {
                 container.addClass("sortable-dragarea");
```

Comment:
I have not looked to much into it, but when I start dragging, all groups are „dimmed“ except the `container`. Not sure how we easily fix this, but not break it for other draggable sortable lists. Maybe we should change the `data-dnd-url` to a global one on the whole table here? Not sure how this then behaves with the category headlines. So maybe live with it or have some other solution to not dim the other groups.